Automattic/wp-calypso
# Plans: Update feature list and FAQs with premium theme info

#### Changes proposed in this Pull Request

* Re-add "Premium themes" as items to the plan features list on the Plans page
* Update copy for themes FAQ on the plans page

**Before**

<img width="1067" alt="Screen Shot 2021-12-09 at 10 19 37 AM" src="https://user-images.githubusercontent.com/2124984/145423912-25cb5ee2-7a85-4b6e-a4ef-49a2421c2c56.png">
<img width="356" alt="Screen Shot 2021-12-09 at 10 19 43 AM" src="https://user-images.githubusercontent.com/2124984/145423905-713bb1db-d0fa-4735-9380-178b6ac9da8b.png">


**After**

<img width="1081" alt="Screen Shot 2021-12-09 at 10 17 12 AM" src="https://user-images.githubusercontent.com/2124984/145423481-77b67d30-e992-4dcc-b40a-8448062b6eb1.png">
<img width="366" alt="Screen Shot 2021-12-09 at 10 16 21 AM" src="https://user-images.githubusercontent.com/2124984/145423485-8bdeee93-c46a-42f6-8f7e-e27099ab4e0b.png">


#### Testing instructions

* Switch to this PR, add `?flags=themes/premium` to the URL to enable the flag
* Visit the Plans page for sites of different types; free, personal, premium, business, ecomm, Jetpack, etc.
* "Premium themes" should appear before "Advanced design customization" in the features list, with a tooltip
* Scrolling down a bit, the FAQ for "Can I install my own themes?" should mention premium themes as well
* These elements should not be visible without the themes/premium flag.
* The copy should match the copy suggested in the marketing spreadsheet.

Related to #58780
